server/views.go

```diff
@@ -53,3 +54,55 @@ func handleAddPlayer(ctx *HandlerCtx) http.HandlerFunc {
 		}
 	}
 }
+
+func handleAddScoreSelectPlayers(ctx *HandlerCtx) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case "GET":
+			players := ctx.DB.GetAllPlayers()
+			ctx.Template.Render(w, players)
+		case "POST":
+			r.ParseForm()
+			addScoreURL, err := url.Parse("/score/add/")
+			if err != nil {
+				panic(err)
+			}
+			q := addScoreURL.Query()
+			for name := range r.Form {
+				q.Add("player", name)
+			}
+			addScoreURL.RawQuery = q.Encode()
+			http.Redirect(w, r, addScoreURL.String(), 302)
```

Comment:
Consider using `http.StatusFound` instead of `302`

Reply:
done